azurerm/resource_arm_storage_account.go

```diff
@@ -1484,6 +1568,27 @@ func flattenStorageAccountVirtualNetworks(input *[]storage.VirtualNetworkRule) [
 	return virtualNetworks
 }
 
+func flattenBlobProperties(input storage.BlobServiceProperties) []interface{} {
+	if input.BlobServicePropertiesProperties == nil {
+		return []interface{}{}
+	}
+
+	blobProperties := make(map[string]interface{})
```

Comment:
due to changes in Terraform 0.12 we need to ensure that all nested fields have a value; as such can we make the `delete_retention_policies` field a top-level field so that we can set this regardless (and conditionally add elements to it)

```suggestion
	deleteRetentionPolicies := make([]interface{}, 0)
```

Reply:
done